themes/bootstrap3/js/searchbox_controls.js

```diff
@@ -62,7 +62,7 @@ VuFind.register('searchbox_controls', function SearchboxControls() {
       $(this).parent().removeClass("active");
     });
     $(".keyboard-selection-item[data-value='" + layoutName + "']").parent().addClass("active");
-    window.Cookies.set("keyboard", layoutName);
+    window.Cookies.set("keyboard", layoutName, {sameSite: 'Lax'});
```

Comment:
I think this makes sense. We should follow the configured sameSite setting, though, but since it's only available when cart is active, it would need a bit more refactoring.

Reply:
Seems like it would be reasonable to expose the sameSite setting universally since any cookie-generating code will need it.

Reply:
Exactly. And not retrieve it via cart in the PHP side either. The js side would fit in cookie.js.

Reply:
Would it make sense for @ThoWagen to incorporate that change here, or should one of us tackle it as a separate PR that becomes a prerequisite to this one?

Reply:
I looked into it and think handling the cookies in cookies.js is a good idea. I will create a separate PR and move the cookie part of this PR there.

Reply:
Here is the separate PR #3941 and this PR now only fixes the keyboard selection.